include/psa/crypto_adjust_keypair_types.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * \file psa/crypto_adjust_keypair_types.h
```

Comment:
Non-blocker: I think the name should be `crypto_adjust_config_key_pair_types.h`. With `config` in the name which I forgot for `crypto_adjust_auto_enabled.h`, and with an underscore in `key_pair` because that's how we spell it in macro names.